conf/messages

```diff
@@ -495,7 +495,7 @@ supporting-evidence.scan-failed.bullet1=the file had a virus or was corrupt
 supporting-evidence.scan-failed.bullet2=you did not select a file to upload
 supporting-evidence.scan-failed.bullet3=you uploaded the wrong type of file
 supporting-evidence.scan-failed.bullet4=you uploaded a file larger than 10MB
-supporting-evidence.scan-failed.p2=We only accept these file types: JPG, PNG, PDF, DOC, and XLS
+supporting-evidence.scan-failed.p2=We only accept these file types: JPG, PNG, PDF
```

Comment:
should there be a period at the end of this sentence? and don't we need an `and` before the `PDF`?

Reply:
There wasn't a period at the end of the original string, but yes, there needs to be an "and"

Reply:
I've put a do not merge on this until the upscan-config PR goes in.